commerce-add-ons/packages/product-feeds/README.md

```diff
@@ -0,0 +1,38 @@
+---
+description: >-
+  Need to generate umbraco commerce product feed? There's a plugin for that.
+---
+
+## Introduction
+After building up your e-commerce website, it's quite common that you want to share your fancy products to a external service like Google Merchant Center to reach out to more viewers.
+
+We have a XML feed generator plugin which simplifies the creation of structured XML files like that. It automatically extracts product data from Umbraco node, organizes it according to standards and allows you to add more information to the feed.
+
+Checkout `Umbraco Commerce Product Feeds plugin`.
+
+Supported feed templates
+- Google Merchant Center.
+
+## Quick start
+- Install the package from nuget: [![NuGet](https://img.shields.io/nuget/v/Umbraco.Commerce.ProductFeeds.svg?style=modern&label=nuget)](https://www.nuget.org/packages/Umbraco.Commerce.ProductFeeds/) 
+
+- To add this plugin to your website, locate where you call `IUmbracoBuilder.AddUmbracoCommerce()` and add a call to `IUmbracoCommerceBuilder.AddCommerceProductFeeds()`.
```

Comment:
just to have more info on where this might be, added a link to https://docs.umbraco.com/umbraco-commerce/key-concepts/umbraco-commerce-builder#registering-dependencies 

Also, we start with actions first when giving instructions
```suggestion
2. Locate where you [register the dependencies](https://docs.umbraco.com/umbraco-commerce/key-concepts/umbraco-commerce-builder#registering-dependencies) `IUmbracoBuilder.AddUmbracoCommerce()` and add a call to `IUmbracoCommerceBuilder.AddCommerceProductFeeds()` to add this plugin to your website. 
```